ops/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,3 @@
+---
+remote_user: "ubuntu"
```

Comment:
This is defaulting to a non-existent user by default. How will ansible run this when it is being installed by a package where you can't pass arguments?